boards/BMS/Src/Io/Io_LTC6813/Io_CellVoltages.c

```diff
@@ -176,7 +190,7 @@ bool Io_NewCellVoltages_ReadVoltages(void)
         {
             // Prepare the command used to read data back from a register group
             uint16_t tx_cmd[NUM_OF_CMD_WORDS] = { 0U };
-            Io_NewLTC6813_PrepareCmd(tx_cmd, cv_read_cmds[curr_reg_group]);
+            Io_SharedLTC6813_PrepareCmd(tx_cmd, cv_read_cmds[curr_reg_group]);
 
             // Transmit the command and receive data stored in register group
             if (Io_SharedSpi_TransmitAndReceive(
```

Comment:
Could we possibly wrap `Io_SharedSpi_TransmitAndReceive()` into `Io_SharedLTC6813`? e.g. `Io_SharedLTC6813_SendReadCmd()`. That way, we won't need to expose the SPI handle outside of `Io_SharedLTC6813.c`

Reply:
Yeah im thinking about this too. We can actually get rid of shared spi entirely and implement SPI commands within the IO_SharedLTC file since the BMS is the only board that uses it. Old PDM needed the SPI driver, but new PDM no longer needs a SPI interface. Lets leave this for another PR though. I think its out of scope rn 🅿️🅿️🅿️

Reply:
LOL